ouroboros-consensus/src/Ouroboros/Consensus/Node/Tracers.hs

```diff
@@ -287,6 +287,17 @@ data TraceForgeEvent blk
     -- Mempool. This will be followed by TraceForgedBlock.
   | TraceNodeIsLeader SlotNo
 
+    -- | We ticked the ledger state for the slot of the to-be-forged block.
+    --
+    -- We record the current slot number and that of the new block.
```

Comment:
> and that of the new block

This is not what we want (it's always the current block). We have a GitHub comment elsewhere about the code for that, but this Haddock needs to updated too.